driver-core/pom.xml

```diff
@@ -211,6 +211,8 @@
                     <resource>META-INF/maven/io.netty/netty-handler/pom.xml</resource>
                     <resource>META-INF/maven/io.netty/netty-transport/pom.properties</resource>
                     <resource>META-INF/maven/io.netty/netty-transport/pom.xml</resource>
+                    <resource>META-INF/maven/io.netty/netty-transport-native-epoll/pom.properties</resource>
```

Comment:
I am adding these two lines to prevent native-epoll pom to show up in the shaded artifact. I realized that actually native-epoll is being shaded too, even if it is optional.